Bottom: a514a2c5ef41789d675ff66e188d8f8a8dcf6e18
Top:    38550d3077df752dba747ce443d59cb405ecd355
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-04 15:57:59 +0000

Adjust circular approximation


---

diff --git a/src/polygon.h b/src/polygon.h
index a02fd99..f2eecdd 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,14 +34,14 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 /* adjustment to make the segments outline the circle rather than connect
  * points on the circle: 1 - cos (\alpha / 2) < (\alpha / 2) ^ 2 / 2
  */
-#define POLY_CIRC_RADIUS_ADJ (1.0 + M_PI / POLY_CIRC_SEGS_F * \
-                                    M_PI / POLY_CIRC_SEGS_F / 2.0)
+#define POLY_CIRC_RADIUS_ADJ 1.0 //(1.0 + M_PI / POLY_CIRC_SEGS_F * \
+                                 //   M_PI / POLY_CIRC_SEGS_F / 2.0)
 
 /* polygon diverges from modelled arc no more than MAX_ARC_DEVIATION * thick */
 #define POLY_ARC_MAX_DEVIATION 0.02
